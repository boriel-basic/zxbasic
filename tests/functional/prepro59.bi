
#define ASCIIZ(x)  (x + CHR$(0))

LET a$ = ASCIIZ(a$)

