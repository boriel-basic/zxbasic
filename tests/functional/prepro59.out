#line 1 "prepro59.bi"



LET a$ =  (a$ + CHR$(0))

